language/es6.json

```diff
@@ -13,6 +13,7 @@
     "no-useless-computed-key": "error",
     "no-useless-concat": "error",
     "prefer-template": "error",
+    "template-curly-spacing": "error",
```

Comment:
This is a "positive" rule (not starting with `no-`) that triggers a test failure if used wrongly and should be covered in the positiveFailures files, except we don't test language rules yet. Meh.